src/Microsoft.AspNetCore.Http/Features/HttpResponseTrailersFeature.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    public class HttpResponseTrailersFeature : IHttpResponseTrailersFeature
```

Comment:
Where do you expect to need this implementation? I expect the servers to provide their own.

Reply:
> Where do you expect to need this implementation?

I don't. I just saw the other features had "dummy" implementations, so just followed along.